base64ct/src/lib.rs

```diff
@@ -263,34 +263,25 @@ pub fn decode_vec(input: &str, padded: bool) -> Result<Vec<u8>, Error> {
 }
 
 /// Get the length of the output from decoding the provided Base64-encoded input.
+///
+/// Note that this function does not fully validate the Base64 is well-formed
+/// and may return incorrect results for malformed Base64.
+///
+/// It is designed to be used in conjunction with [`decode`].
```

Comment:
Is it worth to keep this function public? Previously it could've been used for creating arrays, but now it's no longer `const`.

Reply:
It's still useful for slicing output buffers

Reply:
What do you mean? The `decode` function already returns a sliced buffer.

Reply:
Aah yeah, good point. I take back my comment: we could hide this function from the public API.